syntax = "proto3";

package test.v1;

// Simple test service with basic functionality
message SimpleRequest {
  string name = 1;
  int32 value = 2;
}

message SimpleResponse {
  string result = 1;
  bool success = 2;
}

service SimpleService {
  rpc DoSomething(SimpleRequest) returns (SimpleResponse) {}
}